integrationtests/src/main/java/io/kroxylicious/test/kafkacluster/KafkaClusterExecutionMode.java

```diff
@@ -4,7 +4,7 @@
  * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-package io.kroxylicious.proxy.testkafkacluster;
+package io.kroxylicious.test.kafkacluster;
```

Comment:
I wonder if the package should be:

```suggestion
package io.kroxylicious.testSupport.kafkacluster;
```

just to make it explicit that it is "production code"

Reply:
Perhaps we go so far as to put the `KafkaCluster` code into `io.kroxy.cluster` and the extension into `io.kroxy.junit5` as separate modules in a new repo. 

Also: Package names are supposed to be strictly lower case for deeply historical reasons.

Reply:
both your suggested package names are a lot better than `testSupport` 😀